OS X build script fails on missing sphinx module.

## Summary

The OS X build script fails because of a missing `sphinx_rtd_theme` module.

## Branch method

 - Branches: `installers/develop` and `ka-lite/0.16.x`


## Traceback

http://dungeon.learningequality.org/download/KL-MT16-JOB1/build_logs/KL-MT16-JOB1-102.log

```
...
build	28-Mar-2016 18:18:02	/Applications/Xcode.app/Contents/Developer/usr/bin/make -C docs html
build	28-Mar-2016 18:18:02	sphinx-build -b html -d _build/doctrees   . _build/html
build	28-Mar-2016 18:18:03	Running Sphinx v1.4
build	28-Mar-2016 18:18:03	making output directory...
error	28-Mar-2016 18:18:03	
error	28-Mar-2016 18:18:03	Exception occurred:
error	28-Mar-2016 18:18:03	  File "conf.py", line 118, in <module>
error	28-Mar-2016 18:18:03	    import sphinx_rtd_theme
error	28-Mar-2016 18:18:03	ImportError: No module named sphinx_rtd_theme
error	28-Mar-2016 18:18:03	The full traceback has been saved in /var/folders/dg/2n65wls9481dcx7yvkyz1ck80000gn/T/sphinx-err-_YGPh2.log, if you want to report the issue to the developers.
error	28-Mar-2016 18:18:03	Please also report this if it was a user error, so that a better error message can be provided next time.
error	28-Mar-2016 18:18:03	A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
error	28-Mar-2016 18:18:03	make[1]: *** [html] Error 1
error	28-Mar-2016 18:18:03	make: *** [docs] Error 2
build	28-Mar-2016 18:18:03	.. Abort!  Error/s encountered running 'make dist'.
...
```


## How to reproduce

1. Go to Bamboo server and re-run the build plan at http://dungeon.learningequality.org/browse/KL-MT16 and look at the logs.

--- a/osx/build.sh
+++ b/osx/build.sh
@@ -270,6 +270,16 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+PIP_CMD="$PYRUN_PIP install -r requirements_sphinx.txt"
+# TODO(cpauya): Streamline this to pip install only the needed modules/executables for `make dist` below.
+cd "$KA_LITE_DIR"
+echo ".. Running $PIP_CMD..."
+$PIP_CMD
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Error/s encountered running '$PIP_CMD'."
+    exit 1
+fi
+
 
 ((STEP++))
 echo "$STEP/$STEPS. Running 'make dist'..."

--- a/osx/post_installation.sh
+++ b/osx/post_installation.sh
@@ -13,11 +13,12 @@
 # 4. Run shebangcheck script that checks the python/pyrun interpreter to use.
 # 5. Remove the old asset folder to be replaced by newer assets later.
 # 6. Run kalite manage syncdb --noinput.
-# 8. Run kalite manage setup --noinput.
-# 7. Run kalite manage retrievecontentpack local en path-to-en.zip.
-# 9. Change the owner of the ~/.kalite/ folder and .plist file to current user.
-# 10. Set the KALITE_PYTHON env var for the user doing the install so we don't need to restart after installation.
-# 11. Create a copy of ka-lite-remover.sh and name it as KA-Lite_Uninstall.tool.
+# 7. Run kalite manage setup --noinput.
+# 8. Run kalite manage collectstatic --noinput.
+# 9. Run kalite manage retrievecontentpack local en path-to-en.zip.
+# 10. Change the owner of the ~/.kalite/ folder and .plist file to current user.
+# 11. Set the KALITE_PYTHON env var for the user doing the install so we don't need to restart after installation.
+# 12. Create a copy of ka-lite-remover.sh and name it as KA-Lite_Uninstall.tool.
 
 
 #----------------------------------------------------------------------
@@ -26,7 +27,7 @@
 SCRIPTPATH=$( cd $(dirname $0) ; pwd -P )
 
 STEP=0
-STEPS=11
+STEPS=12
 
 KALITE_SHARED="/Applications/KA-Lite/support"
 KALITE_DIR="$HOME/.kalite"
@@ -204,6 +205,11 @@ $BIN_PATH/kalite manage syncdb --noinput
 ((STEP++))
 msg "$STEP/$STEPS. Running kalite manage setup --noinput..."
 $BIN_PATH/kalite manage setup --noinput
+
+
+((STEP++))
+msg "$STEP/$STEPS. Running kalite manage collectstatic --noinput..."
+$BIN_PATH/kalite manage collectstatic --noinput
 
 
 # Use `kalite manage retrievecontentpack local en path-to-en.zip`.

